README.md

```diff
@@ -1,5 +1,23 @@
 # gha-mergify-ci
 
-GitHub Action to automatically uploads JUnit XML reports to Mergify CI Insights.
+GitHub Action to:
+
+* junit-process: process JUnit XML files with Mergify CI Insights (Upload and Quarantine)
+* scopes: detect and upload pull requests scopes to Mergify Merge Queue
 
 More information on https://mergify.com
+
+## Inputs
+
+<!-- AUTO-DOC-INPUT:START - Do not remove or modify this section -->
+
+|        INPUT         |  TYPE  | REQUIRED |           DEFAULT           |                                                                                                DESCRIPTION                                                                                                |
+|----------------------|--------|----------|-----------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+|        action        | string |  false   |      `"junit-process"`      | The Mergify CI action: * junit-process: <br>process JUnit XML files with Mergify <br>CI Insights (Upload and Quarantine) * scopes: detect <br>and upload pull requests scopes to <br>Mergify Merge Queue  |
```

Comment:
a too bad the rendering is not ideal here, i guess it's not big deal 
<img width="388" height="168" alt="Image" src="https://github.com/user-attachments/assets/307700d6-6635-4db8-b58a-8088adf81f2a" />